docs/swarm/swarm-mode.md

```diff
@@ -0,0 +1,155 @@
+<!--[metadata]>
++++
+title = "Run Docker Engine in swarm mode"
+description = "Run Docker Engine in swarm mode"
+keywords = ["guide, swarm mode, node"]
+advisory = "rc"
+[menu.main]
+identifier="initialize-swarm-guide"
+parent="engine_swarm"
+weight=12
++++
+<![end-metadata]-->
+
+# Run Docker Engine in swarm mode
+
+When you first install and start working with Docker Engine (Engine), swarm mode
+is disabled by default. When you're working outside of swarm mode, you create
+images and run containers. If you need a refresher on containers and images, see
+the [Engine user guide](../userguide/intro.md).
```

Comment:
> When you're working outside of swarm mode, you create images and run containers

I'm not sure this gets the intended point across. It makes it sound like inside swarm mode you can't create images or run containers. Maybe say that swarm mode enables new a new concept called services that you interact with through the `docker service` command.